conan/api/model.py

```diff
@@ -25,6 +25,7 @@ def __init__(self, name, url, verify_ssl=True, disabled=False, allowed_packages=
         self.disabled = disabled
         self.allowed_packages = allowed_packages
         self.remote_type = remote_type
+        self.caching = {}
```

Comment:
If this goes ahead, I'd add an `invalidate_cache` method so clearing the cache is not ugly

Reply:
Agree.